src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -606,7 +606,7 @@
       <CustomToolNamespace>System</CustomToolNamespace>
     </EmbeddedResource>
   </ItemGroup>
-  <Import Project="$(ToolsDir)targets\GenerateThisAssemblyCs.targets" />
-  <Import Project="$(ToolsDir)targets\ResolveContract.targets" Condition="'$(OSGroup)' == 'AnyOS'" />
-  <Import Project="$(ToolsDir)targets\NotSupported.targets" Condition="'$(OSGroup)' == 'AnyOS'" />
+  <Import Project="tools\targets\GenerateThisAssemblyCs.targets" />
+  <Import Project="tools\targets\ResolveContract.targets" Condition="'$(OSGroup)' == 'AnyOS'" />
+  <Import Project="tools\targets\NotSupported.targets" Condition="'$(OSGroup)' == 'AnyOS'" />
```

Comment:
Are these changes on purpose?

Reply:
Yes, I did it to keep tools alongwith netcore project, not mandatory since I managed to get the whole folder copied later.. we can choose eitherway.

Reply:
Moved back to their original location.